src/controllers/midi/hss1394controller.cpp

```diff
@@ -191,6 +189,7 @@ void Hss1394Controller::send(QByteArray data) {
     int bytesSent = m_pChannel->SendChannelBytes(
         (unsigned char*)data.constData(), data.size());
 
+    controllerDebug(formatSysexMessage(getName(), data));
     //if (bytesSent != length) {
```

Comment:
Yeah, that line hasn't been modified since you created the file. If the hss1394 library actually does indicate an error somehow, it would be good to handle that.